yt/odin/bin/yt_odin_generate_config/__main__.py

```diff
@@ -281,6 +281,24 @@ def create_odin_checks_config():
                     "total_threshold": 1000000
                 }
             },
+            "query_tracker_yql_liveness": {
+                "check_timeout": 240,
+                "options": {
+                    "soft_query_timeout": 165,
+                },
+            },
+            "query_tracker_chyt_liveness": {
+                "check_timeout": 240,
```

Comment:
Why do we need such long timeouts for chyt? Do you have the same problem there as in yql?

Reply:
I suspect so, but we didn't enable this check in our clusters.
Let me enable it to make a better answer

Reply:
We couldn't enable qt chyt check since chyt_cluster_name is used for both [proxy address](https://github.com/ytsaurus/ytsaurus/blob/main/yt/odin/checks/bin/query_tracker_chyt_liveness/__main__.py#L14) and [cluster settings for a query](https://github.com/ytsaurus/ytsaurus/blob/main/yt/odin/checks/bin/query_tracker_chyt_liveness/__main__.py#L26)

Our proxy address is `http-proxies-lb.dirac.svc.cluster.local.` while cluster name is `dirac`, so it is not possible for us to enable this check as of now

Reply:
Adjusted chyt check to accept `chyt_cluster_address` and set check_timeout to 120 and soft_timeout to 60
Lowered QL timeouts from (120,60) to (60,30)